variables.tf

```diff
@@ -5,11 +5,6 @@ variable "vnet_name" {
 
 variable "resource_group_name" {
   description = "Default resource group name that the network will be created in."
```

Comment:
1. The reason of removing the "default" param is?
2. If we eventually decide to remove the "default" param, then we may need to polish the description of this variable: "Default resource group name..." since there is no default value assigned to it.

Reply:
I assume I can understand the reason for #1 now given you are using a data source. While based on that the legacy description looks like out of date, would you wanna correct the wordings or just remove the description param?